src/components/molecules/TextView/TextView.tsx

```diff
@@ -14,19 +14,39 @@ interface IProps {
 
 const useStyles = makeStyles((theme: Theme) => ({
   root: {
+    position: 'relative',
+    top: '10%',
     display: 'flex',
     flexDirection: 'column',
     height: '100%',
-    justifyContent: 'space-evenly',
+    width: '100%',
+    justifyContent: 'center',
     letterSpacing: 1,
-    padding: `0 ${theme.spacing(6)}px`,
   },
-  border: {
-    borderBottom: `5px solid ${silverSmokeColor}`,
+  headerBase: {
+    width: '70.51%',
+    height: '20.76%',
+    alignSelf: 'center',
   },
-  image: {
-    height: theme.spacing(11),
+  list: {
+    width: '80%',
+    height: '100%',
+    paddingRight: theme.spacing(12),
+  },
+  singleType: {
+    height: '40%',
     width: 'auto',
+    alignSelf: 'center',
+    position: 'relative',
+    bottom: '20%',
+    margin: theme.spacing(2),
+  },
+  headerSingleType: {
+    position: 'relative',
+    bottom: '20%',
+  },
+  headerList: {
+    margin: '0 6% 0 0',
```

Comment:
?

Reply:
in order to handle cases of single type view and list view, I needed to make small adjustments in the header position so it will show the header properly in both cases